README.md

```diff
@@ -14,14 +14,30 @@ These instructions apply to minor and patch version updates. Major versions need
 a customized adaptation.
 
 After the CI is green:
-  * Update the version in the podspec
-  * Add the CocoaPods tag
-    * `git tag CocoaPods-{version}`
-    * `git push origin CocoaPods-{version}`
-  * Push the podspec to SpecsStaging
-    * `pod repo push staging GoogleDataTransport.podspec`
-  * Run Firebase CI by waiting until next nightly or adding a PR that touches `Gemfile`
-  * On google3, copybara and run a global TAP which should kick off automatically after each PR.
+* Update the version in the podspec to match the latest entry in the [CHANGELOG.md](CHANGELOG.md)
```

Comment:
Realized there isn't a CHANGELOG in this repo? Should I add one?

Reply:
It's at https://github.com/google/GoogleDataTransport/blob/main/GoogleDataTransport/CHANGELOG.md, but probably should move to the root.